File: app/src/main/java/dk/itu/moapd/virtualpiano/ui/PianoKey.java
package dk.itu.moapd.virtualpiano.ui;

import android.graphics.RectF;

class PianoKey {

    private final int mSound;
    private final RectF mRectF;
    private boolean mPressed;

    public PianoKey(int sound, RectF rectF) {
        mSound = sound;
        mRectF = rectF;
    }

    public int getSound() {
        return mSound;
    }

    public RectF getRectF() {
        return mRectF;
    }

    public boolean isPressed() {
        return mPressed;
    }

    public void setPressed(boolean pressed) {
        mPressed = pressed;
    }

}

File: app/src/main/java/dk/itu/moapd/virtualpiano/ui/PianoKey.kt
package dk.itu.moapd.virtualpiano.ui

import android.graphics.RectF

class PianoKey(
    private var mSound: Int,
    private var mRectF: RectF) {

    val sound
        get() = mSound
    val rectF
        get() = mRectF
    var isPressed = false

}
